vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -40,4 +40,10 @@ def textual_ems
     end
     h
   end
+
+  def textual_container_groups
+    num_of_container_groups = @record.number_of(:container_groups)
+    label = "Container groups"
+    {:label => label, :image => "container_group", :value => num_of_container_groups}
```

Comment:
@abonas can you change the newly added strings to use gettext there are some examples in other textula helpers,  this one can be changed to ui_lookup(:tables => "container_group") or _('container groups'), can you change others in this PR and in https://github.com/ManageIQ/manageiq/pull/2661